distributions/demo-server/resources/config/api.yaml

```diff
@@ -1,2 +1,2 @@
-baseUrl: https://localhost:3000
-baseApiUrl: https://localhost:8080
+baseUrl: http://localhost:3000
```

Comment:
Why are we undoing this?

Reply:
didn't mean to push that, was testing without https. i reverted it.